tests/CredentialGenerator.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Akeneo\Pim\tests;
+
+use Symfony\Component\Process\Exception\ProcessFailedException;
+
+/**
+ * Aims to generate the couple client/secret id on a PIM installation.
+ *
+ * @author    Alexandre Hocquard <alexandre.hocquard@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class CredentialGenerator
+{
+    /** @var CommandLauncher */
+    private $commandLauncher;
+
+    /**
+     * @param array $config
+     */
+    public function __construct(array $config)
+    {
+        $this->commandLauncher = new CommandLauncher($config);
```

Comment:
inject command launcher instead of config

Reply:
Done, dunno why this preview was not updated.